crates/test_utils/src/main.rs

```diff
@@ -0,0 +1,163 @@
+use std::{
+    env,
+    process::{exit, Command as cmd},
+};
+
+use clap::{arg, command, Parser};
+use router::types::ConnectorAuthType;
+use test_utils::connector_auth::ConnectorAuthenticationMap;
+
+// Just by the name of the connector, this function generates the name of the collection
+// Example: CONNECTOR_NAME="stripe" -> OUTPUT: postman/stripe.postman_collection.json
+fn path_generation(name: &String) -> String {
+    format!("postman/{}.postman_collection.json", name)
+}
+
+#[derive(Parser)]
+#[command(author, version, about = "Postman collection runner using newman!", long_about = None)]
+struct Args {
+    /// Name of the connector
+    #[arg(short, long = "connector_name")]
+    connector_name: String,
+    /// Base URL of the Hyperswitch environment
+    #[arg(short, long = "base_url")]
+    base_url: String,
+    /// Admin API Key of the environment
+    #[arg(short, long = "admin_api_key")]
+    admin_api_key: String,
+}
+
+// runner starts here
+fn main() {
+    let args = Args::parse();
+
+    let connector_name = args.connector_name;
+    let base_url = args.base_url;
+    let admin_api_key = args.admin_api_key;
+
+    // Function calls
+    let collection_path = path_generation(&connector_name);
+    let auth_map = ConnectorAuthenticationMap::new();
+
+    let inner_map = &auth_map.0;
+
+    // Newman runner
+    // Depending on the conditions satisfied, variables are added. Since certificates of stripe have already
+    // been added to the postman collection, those conditions are set to true and collections that have
+    // variables set up for certificate, will consider those variables and will fail.
+
+    let mut newman_command = cmd::new("newman");
+
+    newman_command.arg("run");
+    newman_command.arg(&collection_path);
+
+    newman_command
+        .arg("--env-var")
+        .arg(format!("admin_api_key={admin_api_key}"));
+
+    newman_command
+        .arg("--env-var")
+        .arg(format!("baseUrl={base_url}"));
```

Comment:
You can merge these multiple calls to `arg()` with [`args(["...", "..."])`](https://doc.rust-lang.org/std/process/struct.Command.html#method.args) instead. Same elsewhere.